### PR TITLE
added at risk and new validator flag

### PR DESCRIPTION
**Pull request by Jibz-Mysten**
## Description 

Describe the changes or additions included in this PR.

- Added at-risk and new validator badge
- Updated countdown component to display `now`

Screenshot 2023-03-14 at 12 42 57 AM

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Comment by mystie711** [comment]
Just a few some small deltas to polish,
at_risk_label_delta

**Comment by Jibz-Mysten** [comment]
Screenshot 2023-03-14 at 10 12 36 AM

File: apps/wallet/src/ui/app/shared/countdown-timer/index.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ export function CountDownTimer({
 
     return (
         <div className={timeStyle(styles)}>
-            {label} {timeAgo}
+            {timeAgo === '< 1 sec' ? 'now' : `${label || ''} ${timeAgo}`}
```
Comment (pchrysochoidis) [inline apps/wallet/src/ui/app/shared/countdown-timer/index.tsx:43]: I would prefer `useTimeAgo` to return the correct/expected value

**Review by pchrysochoidis — APPROVED** [review]

File: apps/wallet/src/ui/app/shared/BadgeUI.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            },
+        },
+    }
+);
+
+export interface CountDownTimerProps extends VariantProps<typeof badgeStyle> {
+    label: string;
+}
+
+export function BadgeUI({ label, ...styles }: CountDownTimerProps) {
```
Comment (Jordan-Mysten) [inline apps/wallet/src/ui/app/shared/BadgeUI.tsx:24]: @Jibz-Mysten In general we shouldn't need things like `UI` in the component names, it's implied.

File: apps/wallet/src/ui/app/shared/BadgeUI.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        variants: {
+            variant: {
+                warning: 'bg-issue-light text-issue-dark',
+                success: 'bg-sui/30 text-hero-dark',
+            },
+        },
+    }
+);
+
+export interface CountDownTimerProps extends VariantProps<typeof badgeStyle> {
```
Comment (Jordan-Mysten) [inline apps/wallet/src/ui/app/shared/BadgeUI.tsx:20]: Naming difference between the component and the props

File: apps/wallet/src/ui/app/staking/validators/ValidatorListItem.tsx [hunk 4]
```diff
@@ -1,80 +1,68 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import { formatAddress } from '@mysten/sui.js';
-import cl from 'classnames';
+import { cx } from 'class-variance-authority';
```
Comment (Jordan-Mysten) [inline apps/wallet/src/ui/app/staking/validators/ValidatorListItem.tsx:4]: I actually don't like this helper, which is why I use `clsx` still.

**Review by Jordan-Mysten — COMMENTED** [review]
Missed this earlier, but some small follow-ups

**Pull request merged**